src/Analyzer.JsonRuleEngine/Operators/ExistsOperator.cs

```diff
@@ -38,6 +38,6 @@ public ExistsOperator(bool specifiedValue, bool isNegative)
         /// <param name="tokenToEvaluate">The JToken to evaluate.</param>
         /// <returns>A value indicating whether or not the evaluation passed.</returns>
         public override bool EvaluateExpression(JToken tokenToEvaluate)
-            => (tokenToEvaluate != null) == EffectiveValue;
+            => (tokenToEvaluate != null) == EffectiveValue ^ this.IsNegative;
```

Comment:
I'm not following why a XOR is a good choice for these operations

If EffectiveValue == true, and IsNegative == true, then EffectiveValue ^ this.IsNegative == true (I understand ^ has precendence over ==), but in that case wouldn't we want to check if (tokenToEvaluate != null) == false?